examples/tts/conf/magpietts/magpietts_dc_en.yaml

```diff
@@ -119,6 +119,7 @@ model:
     has_xattn: true
     xa_d_memory: 768
     xa_n_heads: 1
+    xa_d_head: null  # use caution when specifying `xa_d_head` as it will override the default value of d_model // xa_n_heads in Attention class.
```

Comment:
Should we add this to the yaml config? I would let to add less params to the yaml